pkg/readiness/config/config.go

```diff
@@ -48,6 +54,14 @@ func BuildFromEnvVariables(clientSet kubernetes.Interface, isHeadless bool) (Con
 			return Config{}, fmt.Errorf("the '%s' environment variable must be set", hostNameEnv)
 		}
 	}
+
+	logger := &lumberjack.Logger{
+		Filename:   readinessProbeLogFilePath(),
+		MaxBackups: readIntOrDefault(readinessProbeLoggerBackups, 5),
+		MaxSize:    readInt(readinessProbeLoggerMaxSize),
```

Comment:
What happens if MaxSize or MaxAge is 0? will the default be used instead?

Reply:
yes, the library has a set of defaults for all of these values that are used when you specify a zero value